packages/sommelier.rb

```diff
@@ -35,258 +34,296 @@ class Sommelier < Package
+          #!/bin/bash
+          SOMM="$(pgrep -fc sommelier.elf 2> /dev/null)"
+          if [[ "${SOMM}" -gt "0" ]]; then
+            pkill -f sommelier.elf &>/dev/null
+            pkill -F #{CREW_PREFIX}/var/run/sommelier-wayland.pid &>/dev/null
+            pkill -F #{CREW_PREFIX}/var/run/sommelier-xwayland.pid &>/dev/null
+          else
+            exit 0
+          fi
+          if [[ "$(pgrep -fc sommelier.elf 2> /dev/null)" -gt "0" ]]; then
+            echo "sommelier failed to stop"
+            exit 1
+          else
+            echo "sommelier stopped"
+          fi
+        EOF
+
         # restartsommelier
-        system "echo '#!/bin/bash' > restartsommelier"
-        system "echo 'stopsommelier && startsommelier' >> restartsommelier"
+        IO.write 'restartsommelier', <<~EOF
+          #!/bin/bash
+          stopsommelier && startsommelier
+        EOF
       end
     end
   end
 
   def self.install
     Dir.chdir('vm_tools/sommelier') do
-      system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
+      system "DESTDIR=#{CREW_DEST_DIR} samu -C builddir install"
       Dir.chdir('builddir') do
+        FileUtils.mkdir_p %W[#{CREW_DEST_PREFIX}/bin #{CREW_DEST_PREFIX}/sbin #{CREW_DEST_PREFIX}/etc CREW_DEST_HOME]
         FileUtils.mv "#{CREW_DEST_PREFIX}/bin/sommelier", "#{CREW_DEST_PREFIX}/bin/sommelier.elf"
-        system "install -Dm755 sommelier_sh #{CREW_DEST_PREFIX}/bin/sommelier"
-        system "install -Dm755 sommelierd #{CREW_DEST_PREFIX}/sbin/sommelierd"
-        system "install -Dm755 startsommelier #{CREW_DEST_PREFIX}/bin/startsommelier"
+        FileUtils.install 'sommelier_sh', "#{CREW_DEST_PREFIX}/bin/sommelier", mode: 0o755
+        FileUtils.install 'sommelierd', "#{CREW_DEST_PREFIX}/sbin/sommelierd", mode: 0o755
+        FileUtils.install 'startsommelier', "#{CREW_DEST_PREFIX}/bin/startsommelier", mode: 0o755
         FileUtils.ln_sf 'startsommelier', "#{CREW_DEST_PREFIX}/bin/initsommelier"
-        system "install -Dm755 stopsommelier #{CREW_DEST_PREFIX}/bin/stopsommelier"
-        system "install -Dm755 restartsommelier #{CREW_DEST_PREFIX}/bin/restartsommelier"
-        system "install -Dm755 sommelierrc #{CREW_DEST_PREFIX}/etc/sommelierrc"
-        system "install -Dm644 .sommelier-default.env #{CREW_DEST_HOME}/.sommelier-default.env"
+        FileUtils.install 'stopsommelier', "#{CREW_DEST_PREFIX}/bin/stopsommelier", mode: 0o755
+        FileUtils.install 'restartsommelier', "#{CREW_DEST_PREFIX}/bin/restartsommelier", mode: 0o755
+        FileUtils.install 'sommelierrc', "#{CREW_DEST_PREFIX}/etc/sommelierrc", mode: 0o644
       end
     end
 
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d"
-    @sommelierenv = <<~SOMMELIERENVEOF
-      # Sommelier loading code
-      set -a
-      source ~/.sommelier-default.env
-      source ~/.sommelier.env
-      set +a
-    SOMMELIERENVEOF
     IO.write("#{CREW_DEST_PREFIX}/etc/env.d/sommelier", @sommelierenv)
```

Comment:
Can we add `/usr/local/etc/bash.d/sommelier` and insert `startsommelier` so it autostarts on login?  This removes the need to add it to `~/.bashrc` and the autostart goes away after removing the package.

Reply:
Done, and binaries have been updated.